MyMovies/Views/MovieTableViewCell.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  MovieTableViewCell.swift
+//  MyMovies
+//
+//  Created by Cora Jacobson on 8/15/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+protocol MovieCellDelegate: class {
+    func didUpdateMovie(movie: Movie)
+}
+
+class MovieTableViewCell: UITableViewCell {
+
+    // MARK: - Properties
+    
+    static let reuseIdentifier = "MyMovieCell"
```

Comment:
I like doing it this way, Johnny had a nice idea with this (I think he's the one that put the reuseIdentifier in the cell class as a static constant)